target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ board_config_update
 board="$(cat /tmp/sysinfo/board_name)" 2>/dev/null
 
 case "$board" in
+pc-engines-apu2)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	ucidef_set_interface_raw "opt" "eth2"
```

Comment:
IMO should be wan (eth0) and land (eth1/eth2).  pfSense is a different firmware, we do not need to stick to their rules.